src/messages.js

```diff
@@ -1,9 +1,11 @@
 export const MESSAGE = {
     browserDisconnected:            'The {userAgent} browser disconnected. This problem may appear when a browser hangs or is closed, or due to network issues.',
+    browsersDisconnected:           'The following browsers disconnected: {userAgents}. This problem may appear when a browser hangs or is closed, or due to network issues.',
     cantEstablishBrowserConnection: 'Unable to establish one or more of the specified browser connections. This can be caused by network issues or remote device failure.',
     cantFindBrowserForAlias:        'Cannot find a corresponding browser for the following alias: {alias}.',
     browserNotSet:                  'No browser selected to test against. Use the Runner.browsers() method to specify the target browser.',
     testSourcesNotSet:              'No test file specified. Use the Runner.src() function to specify one or several test files to run.',
+    testsAreNotFound:               'No tests were found. Either the test files contain no tests or the filter function is too restrictive.',
```

Comment:
I like the name `noTestsToRun`, and I'd use it as the first sentence.
`No tests to run. Either the test files contain no tests or the filter function is too restrictive.`